### ISSUE TITLE
Autoprefixer ignores browserslist from package.json

### ISSUE THREAD

**Issue by vfeskov** [postcss/autoprefixer#962]
Seems that autoprefixer ignores `browserslist` config in `package.json` when used like this:
```js
postcss([autoprefixer]).process(css)
// or 
postcss([autoprefixer()]).process(css)
```

Consider this code:

`./scripts/build.js`:
```js
const postcss  = require('postcss')
const autoprefixer = require('autoprefixer')

const browsersFromPackageJson = require('../package.json').browserslist
console.log(`browserslist from package.json:\n${browsersFromPackageJson}\n`)

const defaultNoBrowsers = autoprefixer()
const withBrowsers = autoprefixer({
  browsers: browsersFromPackageJson
})

console.log(`Infos equal: ${withBrowsers.info() === defaultNoBrowsers.info()}\n`)

const css = `a {
  align-items: center;
  display: flex;
  justify-content: center;
}`

Promise.all([
  postcss([withBrowsers]).process(css),
  postcss([defaultNoBrowsers]).process(css)
]).then(([r1, r2]) => {
  console.log(`withBrowsers result:\n${r1.css}\n`)
  console.log(`defaultNoBrowsers result:\n${r2.css}\n`)
  console.log(`Results equal: ${r1.css === r2.css}`)
})
```

Output:
```
browserslist from package.json:
ie >=10,edge >=15,firefox >=28,chrome >=21,safari >=6.1,opera >=15,ios >=7.1,android >=4.4,> 0.5%

Infos equal: true

withBrowsers result:
a {
  -webkit-box-align: center;
  -webkit-align-items: center;
      -ms-flex-align: center;
          align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
      -ms-flex-pack: center;
          justify-content: center;
}

defaultNoBrowsers result:
a {
  -webkit-box-align: center;
      -ms-flex-align: center;
          align-items: center;
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
      -ms-flex-pack: center;
          justify-content: center;
}

Results equal: false
```

Am I missing something? Please help

**Comment by ai** [comment]
You must to pass `opts.from` to `PostCSS#process` with current file part, so Autoprefixer will know file location to load config.

Maybe I need to add warning about `opts.from` to PostCSS in next year.

**Comment by ai** [comment]
I released PostCSS 6.0.15 with warning on missed `from`
